### PR TITLE
Pretty URLs and updated index.php

### PR DESCRIPTION
**Pull request by 55sketch**
did .htaccess to remove .php, wrote function to use ‘pet-details/$id’
rather than the ? syntax, tidied up index to use print_tag function

### CODE REVIEW

File: app/functions/functions.php [hunk 1]
```diff
@@ -130,3 +132,8 @@ function submit_lost(){
 function print_tag($var, $default = '&mdash;') {
 	echo isset($var) ? $var : $default;
 }
+
+function parseUrl() {
+// Trim URL, Sanitize and explode to array at each /
+	return $url = explode('/pet-details/', filter_var(rtrim($_SERVER['REQUEST_URI'], '/'), FILTER_SANITIZE_URL));
+}
```
Comment (DaveOrDead) [inline app/functions/functions.php:141]: Some weird formatting here. Should both lines be 4 spaces?

Comment (55sketch) [inline app/functions/functions.php:141]: haha I'd _literally_ just changed that and about to hit commit

File: app/partials/header.php [hunk 2]
```diff
@@ -39,11 +39,11 @@
                         </li>
                         <li class="c-nav__item <?php if ($page=='lost')
               echo 'is-current'; ?>">
```
Comment (DaveOrDead) [inline app/partials/header.php]: Can you smash these echos on the same line while you're patching up this file?

File: app/functions/functions.php (hunk 1 above)
Comment (DaveOrDead) [inline app/functions/functions.php:141]: Actually everything here looks like 6 spaces. Smells of SO copy and paste to me ;P

File: app/functions/functions.php [hunk 3]
```diff
@@ -38,7 +38,7 @@ function get_posts(){
 			}
```
Comment (DaveOrDead) [inline app/functions/functions.php]: Should indentation be 4 spaces file wide?

Comment (55sketch) [inline app/functions/functions.php]: a) no SO copy and paste THANK YOU, b) I use tabs, and they're set to 4

Comment (DaveOrDead) [inline app/functions/functions.php]: So how come they render as either 7 or 8?

Comment (55sketch) [inline app/functions/functions.php]: I don't believe they do

File: app/functions/functions.php [hunk 4]
```diff
@@ -127,6 +129,13 @@ function submit_lost(){
...
 
+// Print out variable if it exists, replace with default if not
 function print_tag($var, $default = '&mdash;') {
 	echo isset($var) ? $var : $default;
 }
+
+// Pretty URLs
+function parseUrl() {
+	// Trim URL, Sanitize and explode to array at each /
+	return $url = explode('/pet-details/', filter_var(rtrim($_SERVER['REQUEST_URI'], '/'), FILTER_SANITIZE_URL));
```
Comment (DaveOrDead) [inline app/functions/functions.php]: They definitely do. For example, after line 138, line 139 starts level with the 'n' in function. 7 spaces in

Comment (DaveOrDead) [inline app/functions/functions.php]: For posterity turns out its GitHubs way of handling tabs vs spaces. Looks like 7 on mobile and 3 on desktop.

**Comment by DaveOrDead** [comment]
LGTM

**Pull request merged**